docs/docs/conceptual/choosing-a-cms.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Choosing Your CMS(s)
```

Comment:
Should this have an 'e'...? CMSes does 🤷‍♀️

```suggestion
title: Choosing Your CMS(es)
```

Reply:
Ha! I actually wasn't sure, but looking at it now I'm seeing a discussion in Slack in June 2019 where we agreed on `CMSs` so will keep this way.